runtime/doc/lsp.txt

```diff
@@ -413,6 +413,29 @@ For the format of the response message, see:
 For the format of the notification message, see:
     https://microsoft.github.io/language-server-protocol/specifications/specification-current/#notificationMessage
 
+                                                                *on-list-handler*
+
+on_list handler accepts single parameter options which can be passed directly
+to vim.fn.setqflist or vim.fn.setloclist. E.g.:
```

Comment:
I think instead of telling users what they can do with the payload it should first elaborate what's part of the payload. E.g.:

```suggestion
`on_list` receives a table with:

 - `items` table[], structured like |setqflist-what|
 - `title` string, title for the list.
 - `context` table|nil. `ctx` from |lsp-handler|

This table can be used with vim.fn.setqflist or vim.fn.setloclist. E.g.:
```